product-scenarios/1-manage-public-partner-private-apis/1.5-expose-api-to-the-app-developer/1.5.2-expose-api-to-subset-of-app-developer-restricted-by-roles/src/test/java/org/wso2/am/scenario/tests/rest/api/restrictedVisibility/RestApiVisibilityRestrictedByRolesTestCase.java

```diff
@@ -129,9 +116,9 @@ public void testVisibilityOfAPISRestrictedByMultipleRoles() throws Exception {
         apiName = "APIWildCardApi";
         apiContext = "/AddApiWildCardApi";
 
-        createRole(userManagementClient, subscribeRole);
-        createRole(userManagementClient, creatorRole);
-        createUser(userManagementClient, userName, password, subscribeRole);
+        createRole(ADMIN_LOGIN_USERNAME, ADMIN_PASSWORD, subscribeRole);
+        createRole(ADMIN_LOGIN_USERNAME, ADMIN_PASSWORD, creatorRole);
+        createUser(userName, password, new String[]{subscribeRole} , ADMIN_LOGIN_USERNAME, ADMIN_PASSWORD);
```

Comment:
Please remove the unwanted space.

Reply:
Fixed with 7c34999436a66795d70717eb0f074b2907bcd6bf